src/MutableIterator.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ *
+ * @category   Humbug
+ * @package    Humbug
+ * @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+ * @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+ * @author     Thibaud Fabre
+ */
+namespace Humbug;
+
+use Symfony\Component\Finder\Finder;
+use Traversable;
+
+class MutableIterator implements \IteratorAggregate, \Countable
+{
+
+    /**
+     * @var Container
+     */
+    private $container;
+
+    /**
+     * @var Finder
+     */
+    private $finder;
+
+    /**
+     * @var array
+     */
+    private $mutables = null;
+
+    /**
+     * @param Container $container
+     * @param Finder $finder
+     */
+    public function __construct(Container $container, Finder $finder)
+    {
+        $this->container = $container;
+        $this->finder = $finder;
+    }
+
+    /**
+     * @return array
+     */
+    protected function getMutables()
+    {
+        if ($this->mutables === null) {
+            $this->mutables = $this->container->getMutableFiles($this->finder);
+        }
+
+        return $this->mutables;
+    }
+
+    /**
+     * (PHP 5 &gt;= 5.0.0)<br/>
+     * Retrieve an external iterator
+     *
+     * @link http://php.net/manual/en/iteratoraggregate.getiterator.php
+     * @return Traversable An instance of an object implementing <b>Iterator</b> or
+     * <b>Traversable</b>
+     */
```

Comment:
This comment does not help IMO. 
Leave just `@return Traversable`, or in fact `@return \ArrayIterator`